misc/connection-limits/src/lib.rs

```diff
@@ -92,6 +97,11 @@ impl Behaviour {
     pub fn limits_mut(&mut self) -> &mut ConnectionLimits {
         &mut self.limits
     }
+
+    /// Returns a mutable reference to [`BypassRules`].
+    pub fn bypass_rules_mut(&mut self) -> &mut BypassRules {
```

Comment:
what is the advantage of having an external structure (`BypassRules`) instead of having these methods in the main `Behaviour`?